Makefile-switchroot.am

```diff
@@ -57,6 +57,11 @@ ostree_remount_SOURCES = \
     $(NULL)
 ostree_remount_CPPFLAGS = $(AM_CPPFLAGS) -Isrc/switchroot
 
+if BUILDOPT_SYSTEMD
+ostree_prepare_root_CPPFLAGS += -DHAVE_SYSTEMD=1
+ostree_prepare_root_LDADD = $(LIBSYSTEMD_LIBS)
```

Comment:
I think we need
`ostree_prepare_root_LDADD = $(AM_LDFLAGS) $(LIBSYSTEMD_LIBS)`
to continue inheriting the global flags.